jerry-core/parser/js/js-lexer.c

```diff
@@ -1220,10 +1220,24 @@ lexer_parse_string (parser_context_t *context_p, /**< context */
         continue;
       }
     }
+    else if ((*source_p == LEXER_NEWLINE_LS_PS_BYTE_1 && LEXER_NEWLINE_LS_PS_BYTE_23 (source_p)))
```

Comment:
This can be moved before `else if (str_end_character == LIT_CHAR_GRAVE_ACCENT)`, then the `else if (*source_p == LEXER_NEWLINE_LS_PS_BYTE_1 && LEXER_NEWLINE_LS_PS_BYTE_23 (source_p))` can be removed inside `else if (str_end_character == LIT_CHAR_GRAVE_ACCENT)`